consensus/state.go

```diff
@@ -2474,3 +2474,21 @@ func repairWalFile(src, dst string) error {
 
 	return nil
 }
+
+const SyncDataInterval = 100
+
+// sync data periodically checks to make sure that all block parts in the data
+// routine are pushed through to the state.
+// TODO remove nolint
+//
+//nolint:unused
+func (cs *State) syncData() {
+	for {
+		select {
+		case <-cs.Quit():
+			return
+		case <-time.After(time.Millisecond * SyncDataInterval):
+			// TODO: implement then call
+		}
+	}
+}
```

Comment:
because we're calling this here, I'm not sure we should also import the consensus reactor in the propagation reactor. One thing we could do is just use an interface. I'm not picky about it tho

Reply:
Good idea. Ill change this if there is any circular dependency issue. Thanks